apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -158,18 +207,27 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
   def handle_event("publish", _, socket) do
     ingestion_id = socket.assigns.ingestion.id
 
+    save_ingestion_safe(socket)
```

Comment:
Save the changeset on the socket before messaging all the subforms to mass-save to the DB